Doc/release-notes/master.txt

```diff
@@ -15,8 +15,10 @@ Updates since v1.4.0
  * Update libtiff from 4.0.6 to 4.1.0
  * Update ZLib from 1.2.9 to 1.2.11
  * Update Eigen from 3.3.4 to 3.3.7
+ * Update GTest from 1.8.1 to 1.10.0
  * Add VTK version 9.0.1
 
+
```

Comment:
Unnecessary blank line?

Reply:
Thanks .. fixing. The result of resolving a conflict using github in the browser.